APP/ev_chargers/lib/screens/homepage.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:ev_chargers/style.dart';
+import 'package:flutter/material.dart';
+import 'map_page.dart';
+import 'reservation_page.dart';
+import 'profile_page.dart';
+import 'settings_page.dart';
+
+class MyHome extends StatefulWidget {
```

Comment:
Why not `Home`?

Reply:
Same as before, we are new to flutter and we don't know all the widgets. This is something like our team convention